examples/plot_02_overview_skore_ui.py

```diff
@@ -19,21 +21,28 @@
 import tempfile
 from pathlib import Path
 
-temp_dir = tempfile.TemporaryDirectory(prefix="skore_example_")
-temp_dir_path = Path(temp_dir.name)
-
 # %%
-import subprocess
+# Creating and loading a skore project
+# ====================================
+# %%
+import altair as alt
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import PIL
+import plotly.express as px
+from sklearn.datasets import load_diabetes
+from sklearn.linear_model import Lasso
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
```

Comment:
Please revert these changes.

Reply:
Done.

Reply:
No its not, sorry. Check the diff.

Reply:
My bad I did not understood correctly. It should be done now.